app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -353,6 +353,8 @@ def add_secret_to_pod_def(pod_def, inspector_admin_secret_name)
   end
 
   def inspector_image
-    'docker.io/openshift/image-inspector:v2.0.z'
+    registry = Settings.ems.ems_kubernetes.image_inspector_registry || INSPECTOR_REGISTRY_FALLBACK
+    repo = Settings.ems.ems_kubernetes.image_inspector_repository || INSPECTOR_REPOSITORY_FALLBACK
```

Comment:
@cben @moolitayer also remember that empty strings are different from `nil`:

```ruby
> nil || "fallback value"
=> "fallback value"

> "" || "fallback value"
=> ""
```

You may want to use `blank?`.